README.md

```diff
@@ -2,139 +2,140 @@
-      ```yaml
-         energy:  # field
-            doc:
-               - |
-                 Part1 of the entire doc.
-                 part1 of the entire doc.
-               - |  # Reference for concept
-                 xref:
-                   spec: <spec>
-                   term: <term>
-                   url: <url>"
-               - |
-                 Rest of the doc
-                 rest of the doc
-         velocity:  # field
-            doc: |
-               A single block of doc string.
-      ```
-      Such structure of doc would appear in `nxdl` as
-      ```xml
-      ...
-      <field name="energy">
-        <doc>
-            Part1 of the entire doc.
-            part1 of the entire doc.
-
-                This concept is related to term `&lt;term&gt;`_ of the &lt;spec&gt; standard.
-            .. _&lt;term&gt;: &lt;url&gt;
-
-            Rest of the doc
-            rest of the doc
-        </doc>
-      </field>
-      <field name="velocity">
-        <doc>
-            A single block of doc string.
-        </doc>
-      </field>
-      ```
-
-
-
-* **exists** Options are recommended, required, [min, 1, max, `infty`] numbers like here 1 can be replaced by any `uint` (unsigned integer), or `infty` to indicate no restriction on how frequently the entry can occur inside the NXDL schema at the same hierarchy level.
-* **link** Define links between nodes.
-* **units** A statement introducing NeXus-compliant NXDL units arguments, like NX_VOLTAGE
-* **dimensions** Details which dimensional arrays to expect
-* **dim** Shorthand notation for dimensions, e.g., `(n, )` for an 1D array of length `n` or `(n, m)` for an 2D array of size `n x m`.
-* **enumeration** Python list of strings which are considered as recommended entries to choose from.
-* **dim_parameters** `dim` which is a child of `dimension` and the `dim` might have several attributes `ref`,
-`incr` including `index` and `value`. So while writing `yaml` file schema definition please following structure:
 ```yaml
-dimensions:
-   rank: integer value
-   dim: [[ind_1, val_1], [ind_2, val_2], ...]
-   dim_parameters:
-      ref: [ref_value_1, ref_value_2, ...]
-      incr: [incr_value_1, incr_value_2, ...]
+energy:
+  doc:
+    - |
+      Part1 of the entire doc.
+      part1 of the entire doc.
+    - |
+      xref:
+        spec: <spec>
+        term: <term>
+        url: <url>
+    - |
+      Rest of the doc
+      rest of the doc
+  velocity:  # field
+    doc: |
+        A single block of doc string.
 ```
-Keep in mind that length of all the lists must have the **same size**.
-**Important Note**: The attributes `ref`, `incr`, `index` are deprecated.
 
-## Project roadmap
+This structure appears in Nxdl as follows:
+
+```xml
+<field name="energy">
+  <doc>
+    Part1 of the entire doc.
+    part1 of the entire doc.
+    This concept is related to term `<term>`_ of the `<spec>` standard.
```

Comment:
Isn't there an offset to the **This concept is related to term** line?
Like this:
```xml
<field name="energy">
  <doc>
    Part1 of the entire doc.
    part1 of the entire doc.
        This concept is related to term `<term>`_ of the `<spec>` standard.
   .. _`<term>`: `<url>`
```
If so, please change it here. If this offset does exists, I would suggest to remove it at some point since it doesn't serve any purpose.

Reply:
Ah I see. I always thought that this offset was specifically build in for some rst formatting peculiarities. If not I agree that we should remove entirely.

Reply:
![image](https://github.com/FAIRmat-NFDI/nyaml/assets/50139597/5174e474-40bc-4fba-83ae-265685100037)

I don't see why that offset would help with the rst formatting.

Reply:
Good point. Let's remove it

Reply:
Shall we just remove this offset with this PR as well?

Reply:
Yes, I just committed the change for this